maintainers/maintainer-list.nix

```diff
@@ -5384,6 +5384,12 @@
     githubId = 4158274;
     name = "Michiel Leenaars";
   };
+  legendofmiracles = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add legendofmiracles``.